tests/functional/testServers.js

```diff
@@ -162,6 +162,7 @@ describe('Server list', function() {
   });
 
   it('server switching', async () => {
+    await vpn.setSetting('server-switch-notification', 'true');
```

Comment:
Good! Can you also add 1 line in testSettings.js to test the UI  of this new checkbox?

Reply:
Added :)